src/cohesion_model_washino_capillary_viscous.h

```diff
@@ -269,7 +269,7 @@ namespace ContactModels {
       if(volBond1000 < 1e-14) return;
 
       const double rEff = radi*radj / (radi+radj);
-      const double contactAngleEff = 0.5 * contactAngle[itype] * contactAngle[jtype];
+      const double contactAngleEff = 0.5 * (contactAngle[itype] * contactAngle[jtype]);
```

Comment:
Did you mean a `+` instead of a `*` inside the bracket?

Reply:
Yes. This is an early version of the pull request. I actually submitted two other requests but they only appear in my own LIGGGHTS-PUBLIC repository (fork). I don't know why. If you feel it is necessary to refer to my changes, I will re-submit the pull request (and close this one).

Reply:
yes, please provide a correct patch